erdgo/cmd/testBlock/main.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-sdk/erdgo"
+)
+
+func main() {
+	ep := erdgo.NewElrondProxy("http://174.138.103.62:8079")
```

Comment:
what IP is this?

can you use localhost instead of a specific digital ocean ip?

Reply:
:+1: yes, let's not use this IP, and use some dummy host / localhost instead.

Reply:
Done